backend/app/models/__init__.py

```
@@ -1,5 +1,6 @@
 from app.models.user import User
 from app.models.journal import JournalEntry
 from app.models.pesticide import PesticideProduct
+from app.models.ncpms import NcpmsDiagnosis
 
 __all__ = ["User", "JournalEntry", "PesticideProduct"]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)